packages/crayons-core/src/components/form/readme.md

```diff
@@ -244,6 +244,80 @@ Pass [formSchema](#form-schema) to render Dynamic Form. You can also pass `initi
         placeholder: '',
         multiple: true,
       },
+      {
+        id: 'f319f86f-1b6a-49cb-b4b6-cf4873674595',
```

Comment:
can you also add example where type is passed as a string . And can u add the samples under react section as well.

Reply:
Done.